sycl/include/sycl/ext/oneapi/matrix/matrix-jit-use.hpp

```diff
@@ -39,9 +39,7 @@ SPV_MATRIX_LAYOUT_TRAITS(layout::packed_a, __spv::MatrixLayout::PackedA)
 SPV_MATRIX_LAYOUT_TRAITS(layout::packed_b, __spv::MatrixLayout::PackedB)
```

Comment:
when we should make the change: unused --> dynamic
packed_a and packed_b --> packed